Problem039/C++/Makefile

```diff
@@ -1,7 +1,7 @@
-CXX = g++
```

Comment:
[It](https://github.com/DestructHub/ProjectEuler/blob/master/Problem039/C%2B%2B/solution_1.cpp) is pure c.